packages/token-search-discovery-controller/src/token-discovery-api-service/token-discovery-api-service.ts

```diff
@@ -39,4 +44,60 @@ export class TokenDiscoveryApiService extends AbstractTokenDiscoveryApiService {
 
     return response.json();
   }
+
+  async getTopLosersByChains(
+    topLosersParams: TopLosersParams,
+  ): Promise<MoralisTokenResponseItem[]> {
+    const url = new URL('/tokens-search/top-losers-by-chains', this.#baseUrl);
+
+    if (topLosersParams.chains && topLosersParams.chains.length > 0) {
+      url.searchParams.append('chains', topLosersParams.chains.join());
+    }
+    if (topLosersParams.limit) {
+      url.searchParams.append('limit', topLosersParams.limit);
+    }
+
+    const response = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (!response.ok) {
+      throw new Error(
+        `Portfolio API request failed with status: ${response.status}`,
+      );
+    }
+
+    return response.json();
```

Comment:
Do we need to validate the response in any way or not?

Reply:
I agree with the validation but how do you feel about revisiting this in portfolio api first since it's closest to the server, and bad shapes would never even reach this point?
(this gives more credence to refactoring the error handling in the core repo you mention elsewhere)